bluebottle/test/utils.py

```diff
@@ -49,9 +49,12 @@ def css_dict(style):
 class InitProjectDataMixin(object):
 
     def init_projects(self):
+        from django.core import management
         """
         Set up some basic models needed for project creation.
         """
+        management.call_command('loaddata', 'project_data.json', verbosity=0)
```

Comment:
I think this should already be loaded by migration
https://github.com/onepercentclub/bluebottle/blob/saas/bluebottle/bb_projects/migrations/0003_project_phases_and_themes.py#L12